assets/package.json

```diff
@@ -3,7 +3,8 @@
   "license": "MIT",
   "scripts": {
     "deploy": "webpack --mode production",
-    "watch": "webpack --mode development --watch"
+    "watch": "webpack --mode development --watch",
+    "test": "jest --config ./js/test/jest.conf.js --coverage"
```

Comment:
Curious if there's a way to reference this file path outside of the command?   Jest might by default look for certain config file(s) where we could declare it?